src/GraphMap/Map.cs

```diff
@@ -0,0 +1,289 @@
+using System.Drawing;
+using System.Numerics;
+using System.Text;
+using System.Xml.Schema;
+
+public class Map

```

Comment:
Maybe we should streamline the naming for this class. It is strange to call it `Map` instead of GameMap or something like this. The name `Map` is already occupied with the dynamic data type.

Reply:
You are correct, I changed the name and additionally deleted unused imports.